### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by maksym-nemera**
[DEMO LINK](https://maksym-nemera.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +15,85 @@ export const goods = [
...
-          </td>
-        </tr>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  this.setState({ selectedGood: '' });
+                }}
```
Comment (dinaminahi) [inline src/App.tsx:45]: Let's create a separate function to clear the good and to set selectedGood

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +15,85 @@ export const goods = [
...
+                        data-cy="RemoveButton"
+                        type="button"
+                        className="button is-info"
+                        onClick={() => {
+                          this.setState({ selectedGood: '' });
+                        }}
+                      >
+                        -
+                      </button>
+                    )}
```
Comment (dinaminahi) [inline src/App.tsx:86]:
```suggestion
                  {selectedGood === item
                    ? (
                      <button
                        data-cy="RemoveButton"
                        type="button"
                        className="button is-info"
                        onClick={() => {
                          this.setState({ selectedGood: '' });
                        }}
                      >
                        -
                      </button>
                    ) : (
                       <button
                        data-cy="AddButton"
                        type="button"
                        className="button"
                        onClick={() => {
                          this.setState({ selectedGood: item });
                        }}
                      >
                        +
                      </button>
                    )}
```

**Review by dinaminahi — CHANGES_REQUESTED** [review]
Good job! 👍 
Let's just fix few things to improve readability

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +15,89 @@ export const goods = [
...
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        {!selectedGood
```
Comment (silchyn) [inline src/App.tsx:40]: it's better for understanding to have this condition positive instead of negative (`selectedGood` instead of `!selectedGood`)

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**